kustomize/webhook/validating-webhook.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: admissionregistration.k8s.io/v1beta1
+kind: ValidatingWebhookConfiguration
+metadata:
+  name: terrascan.<TERRASCAN_NAMESPACE>.svc
+  annotations:
+    terrascan-webhook: terrascan.<TERRASCAN_NAMESPACE>.svc
+webhooks:
+  - name: terrascan.<TERRASCAN_NAMESPACE>.svc
+    rules:
+    - apiGroups:
+      - "*"
+      apiVersions:
+        - "*"
+      operations:
+        - CREATE
+        - UPDATE
+      resources:
+        - "*"
+    failurePolicy: Fail
+    sideEffects: None
+    clientConfig:
+      service:
+        name: terrascan
+        namespace: terrascan
```

Comment:
the namespace should be configurable and the same should be mentioned in the document as well

Reply:
thanks for point that out, there was supposed to be the <TERRASCAN_NAMESPACE> placeholder there.. just like line 8. i forgot to add that.